deploy/role.yaml

```diff
@@ -9,14 +9,24 @@ rules:
   resources:
   - configmaps
```

Comment:
I suspect that the deployment instructions need these ConfigMap permissions, but that the operator itself doesn't.

Reply:
I added the verbs for the config map by monitoring the logs of BMO. 
My conclusion is that BMO needs needs the permission on the ConfigMap(s).